components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -175,10 +176,17 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                 >
                     {hasSubmenu ? subMenu!.map((s) => {
                         const hasDivider = s.id === 'ChannelMenu-moveToDivider';
+                        let aria = ariaLabel;
+                        if (s.action) {
+                            aria = s.text === selectedValueText ?
+                                s.text + ' ' + Utils.localizeMessage('sidebar.menu.item.selected', 'selected') :
+                                s.text + ' ' + Utils.localizeMessage('sidebar.menu.item.notSelected', 'not selected');
+                        }
                         return (
                             <span
                                 className={classNames(['SubMenuItemContainer', {hasDivider}])}
                                 key={s.id}
+                                tabIndex={s.id.includes('Divider') ? 1 : 0}
```

Comment:
is this ok to check if something is a divider?

Reply:
Yeah, that should be fine. That's probably about as generic as we can get when checking for those with how the menus currently work